java-logging-appinsights/README.md

```diff
@@ -110,9 +119,15 @@ Along with custom Source name there are more optional headers to be applied at w
 - `SyntheticTest-RunId`
 - `SyntheticTest-Location`
 
+[SyntheticHeaders](java-logging-appinsights/src/main/java/uk/gov/hmcts/reform/logging/appinsights/SyntheticHeaders) might be deprecated in future based on Microsoft making the constants public.
+
+#### To disable Application insights completely 
+
+Refer https://github.com/Microsoft/ApplicationInsights-Java/tree/master/azure-application-insights-spring-boot-starter#completely-disable-application-insights-using-applicationproperties
+
 #### Developer mode
 
-By default developer mode is off. To turn it on set in configuration manually (for telemetry client to pick it up):
+By default developer mode is false. This is controlled by starter and to turn it on set in configuration manually (for telemetry client to pick it up):
```

Comment:
Perhaps the more correct way would be _developer mode is disabled_? The rest is optional change but more explicit 👍

Reply:
Done